app/models/like.rb

```diff
@@ -0,0 +1,16 @@
+class Like < ApplicationRecord
+  belongs_to :user, foreign_key: 'author_id'
+  belongs_to :post
+
+  validates :author_id, presence: true
+  validates :post_id, presence: true
+
+  after_save :update_likes
+
+  private
+
+  def update_likes
+    post = self.post
+    post.update(likes_counter: post.likes.count)
+  end
```

Comment:
- [ ] As a final note here let's make sure to remove the `private` access modifier from the `update_likes` method for same reasons mentioned before. 😉

Reply:
Will work on it according will change it to public